src/Controls/src/Core/HandlerImpl/Editor.Impl.cs

```diff
@@ -4,6 +4,10 @@ public partial class Editor : IEditor
 	{
 		Font? _font;
 
+		TextAlignment _horizontalTextAlignment = TextAlignment.Start;
+
 		Font ITextStyle.Font => _font ??= Font.OfSize(FontFamily, FontSize).WithAttributes(FontAttributes);
+
+		public TextAlignment HorizontalTextAlignment { get { return _horizontalTextAlignment; } set { _horizontalTextAlignment = value; } }
```

Comment:
Is this a new property? Do we need to add a bindable property for this? If we do, then the `Editor` needs to also use the `TextAlignmentElement` for the work as well as implement the `ITextAlignmentElement` interface.

There is a `OnHorizontalTextAlignmentPropertyChanged` method that I have no idea what that does as it is always empty... Might be safe to remove that too. 

I think it is fine to use `TextAlignmentElement` and not implement anything with the vertical part. @PureWeen I am not sure about the rules around this?

Reply:
Good point, removed the new property so now it uses `TextAlignmentElement`.

What should be done with `OnHorizontalTextAlignmentPropertyChanged` and the vertical part?